packages/spotlight/src/electron/main/index.ts

```diff
@@ -100,6 +100,7 @@ app.on("ready", () => {
 
 Sentry.init({
   dsn: "https://192df1a78878de014eb416a99ff70269@o1.ingest.sentry.io/4506400311934976",
+  enabled: !!process.env.NODE_ENV && process.env.NODE_ENV !== "development",
```

Comment:
Why not `process.env.NODE_ENV && process.env.NODE_ENV !== "development"`?

Reply:
because it's inferred as `"" | boolean | undefined` but enabled should be just `boolean | undefined` 🤷

Reply:
You can alternatively do `Boolean(process.env.NODE_ENV)` btw

Reply:
oh, let's do the casting. thanks, much more explicit